modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -520,10 +520,13 @@ def build_supporting_docs(bgs_claim)
                    file_number = local_bgs_service.find_by_ssn(target_veteran.ssn)&.dig(:file_nbr) # rubocop:disable Rails/DynamicFindBy
```

Comment:
We need to replace this call to BGS with [@file_number](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/claims_api/v2/application_controller.rb#L195)

Reply:
Well, after a lengthy discussion with Kayla, I see why this is our temporary solution.